android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -106,32 +117,29 @@ class MusicService : HeadlessJsTaskService() {
 
         notificationCapabilities.forEach {
             when (it) {
-                Capability.PLAY -> {
-                    val playIcon = Utils.getIconOrNull(this, options, "playIcon")
-                    buttonsList.add(PLAY(icon = playIcon))
-                }
-                Capability.PAUSE -> {
-                    val pauseIcon = Utils.getIconOrNull(this, options, "pauseIcon")
-                    buttonsList.add(PAUSE(icon = pauseIcon))
+                Capability.PLAY, Capability.PAUSE -> {
+                    val playIcon = BundleUtils.getIconOrNull(this, options, "playIcon")
+                    val pauseIcon = BundleUtils.getIconOrNull(this, options, "pauseIcon")
+                    buttonsList.add(PLAY_PAUSE(playIcon = playIcon, pauseIcon = pauseIcon))
```

Comment:
@mpivchev is there a possibility of this button getting added twice if the user added both Capability `PLAY` and `PAUSE`? I wrote something on the off chance that was a thing.

https://github.com/doublesymmetry/react-native-track-player/pull/1731/files#diff-5f8591a257348c4875d613bf1dad66a989949b38117fa649c9db24a30cbe0708R110

Reply:
I don't think so. This only triggers an ExoPlayer flag down the line to enable the play/pause button. If it was already set to true, then nothing will change.